src/main/scala/ztis/model/RandomPrediction.scala

```diff
@@ -0,0 +1,33 @@
+package ztis.model
+
+import org.apache.spark.SparkContext._
+import org.apache.spark.mllib.recommendation.Rating
+import org.apache.spark.rdd.RDD
+
+import scala.util.Random
+
+class RandomPrediction(trainingData: RDD[Rating]) extends Predictor with Serializable {
```

Comment:
Yes, without it i've got:
org.apache.spark.SparkException: Task not serializable